metadata-ingestion/src/datahub/ingestion/source/Thrift.g4

```diff
@@ -0,0 +1,222 @@
+grammar Thrift;
```

Comment:
Could you add a comment on this file is defining and if there are any current limitations, including a link to the documentation?

Reply:
OK